### PR TITLE
Public controllers

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| License       | MIT

Requires https://github.com/symfony/symfony/pull/24126

### CODE REVIEW

**Comment by chalasr** [comment]
Merging now would make any controller private, right? Should this wait for the next 3.3 release of HttpKernel?

**Comment by javiereguiluz** [comment]
Can someone please explain this change to me? I can't understand why we remove `public: true` here.

**Comment by xabbuh** [comment]
With the changes done in symfony/symfony#24126, services tagged with `controller.service_arguments` will become public implicitly.

**Comment by javiereguiluz** [comment]
@xabbuh thanks! I totally missed that change!

File: symfony/framework-bundle/3.3/config/services.yaml [hunk 1]
```diff
@@ -21,9 +21,8 @@ services:
         # but if a service is unused, it's removed anyway
         exclude: '../src/{Entity,Repository,Tests}'
 
-    # controllers are imported separately to make sure they're public
-    # and have a tag that allows actions to type-hint services
+    # controllers are imported separately to make sure they have the tag
```
Comment (javiereguiluz) [inline symfony/framework-bundle/3.3/config/services.yaml]: I'm not sure about the need to mention here that this tag internally makes these services public. Would a comment like this be better?

```
# controllers are imported separately to apply the tag that allows actions to type-hint services
```

**Pull request merged**